koch.nim

```diff
@@ -10,7 +10,7 @@
 #
 
 const
-  NimbleStableCommit = "afd03bc000f4013c48f52381728327dabfccfc4c" # master
+  NimbleStableCommit = "7efb226ef908297e8791cade20d991784b4e8bfc" # master
```

Comment:
```suggestion
  NimbleStableCommit = "afd03bc000f4013c48f52381728327dabfccfc4c" # master
```

Reply:
Either bumping nimble or reverting this PR should make CI green.

Reply:
=> https://github.com/nim-lang/Nim/pull/21259